views.py

```diff
@@ -285,20 +216,10 @@ def pokemon_detail(pokemon_name):
 
     user_id = session.get('user_id')
     pokemon = Pokemon.query.filter_by(name=pokemon_name).first_or_404()
-    all_sightings = Sighting.query.order_by(Sighting.sighting_id).all()
-    p_type = ' '.join(pokemon.poke_type)
-
-    seen = 0
-    not_seen = 0
-
-    for row in all_sightings:
-        if row.pokemon_id == pokemon.pokemon_id:
-            seen +=1
-        else:
-            not_seen += 1
-    totals = [seen, not_seen]
+    all_sightings = pokemon.sightings
+    totals = (len(all_sightings), Pokemon.count())
```

Comment:
Again, this only counts if the model suggestion I made turns out to be a good assumption. Otherwise, just get a count using `.distinct()`.